Account for linter complains about terminology (#530)
Azure/ResourceModules: constructs/Microsoft.Management/managementGroup-structure/readme.md

| Parameter Name | Type | Default Value | Possible values | Description |
| :-             | :-   | :-            | :-              | :-          |

> | `name` | string | | | Mandatory. The resource ID of the Management group |

Reviewer: ```suggestion
| `name` | string | | | Required. The group ID of the Management group |
```

> | `parentId` | string | | A MG name | Mandatory. The template will concatenate `/providers/Microsoft.Management/managementGroups/` to create the resource ID of the parent management group the deployed one is child of |

Reviewer: ```suggestion
| `parentId` | string | | A MG name | Required. The template will concatenate `/providers/Microsoft.Management/managementGroups/` to create the resource ID of the parent management group the deployed one is child of |
```

> **Management Group Contributor** at the top management group that needs to be managed
>
>> Consider using the following script:<br>

> >> `$PrincipalID = "<The resource ID of the identity here>"`<br>
> >> `$TopMGID = "<The resource ID of the management group here>"`<br>

Reviewer: ```suggestion
>> `$PrincipalID = "<The object ID of the identity here>"`<br>
>> `$TopMGID = "<The group ID of the management group here>"`<br>
```
